blocks/hdr/hdr.js

```diff
@@ -0,0 +1,42 @@
+import { getLibs } from '../../scripts/utils.js';
+
+function getMediaFilename(a) {
+  try {
+    const mediaUrl = new URL(a.href);
+    return mediaUrl.pathname;
+  } catch (e) {
+    console.log('Error parsing media url', e);
+  }
+  return '';
+};
+
+export default async function init(el) {
+  const links = el.querySelectorAll(':scope a');
```

Comment:
I assume if the block is being imported then it should always have at least one link?

If that's the case I think it's ok to import utils at top of file.

Reply:
True.